lib/python/rose/config_tree.py

```diff
@@ -485,7 +485,7 @@ def run(self):
 if __name__ == "__main__":
     # These modules are only required for running the self tests.
     from rose.config import ConfigDumper
-    from StringIO import StringIO
+    from io import StringIO
     from shutil import rmtree
     from tempfile import mkdtemp
     _Test().run()
```

Comment:
Move to unittest or pytest in the future.

Reply:
#2308